styles/main.css

```diff
@@ -43,32 +44,44 @@ p {
 a {
 	color:orange;
 }
+nav ul {
+    float: right;
+    margin: 0;
+    padding: 0;
+}
 nav ul li{
-	display: inline-block;
+    margin: 0;
 	list-style-type: none;
         float:left;
 	
 }
 nav ul li a {
 	color:#ff00ff;
 	text-decoration: none;
+    
+    display: block;
+    padding-top: 1em;
+    padding-bottom: 0.5em;
+    margin: 0 1em;
 	
+    border-top: 4px solid transparent;
+}
+nav ul li a:hover {
+    border-top: 4px solid black;
 }
 header{
-	    border-bottom: 4px solid rgba(0,0,0.1);
-    padding: 2em;
+    border-top: 4px solid rgba(0,0,0,0.1);
```

Comment:
The header wast to have a TOP border.
You are also missing one value in the rgba colour